.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,106 @@
+name: Run Tests
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      source:
+        description: 'Source of tests'
+        required: true
+        type: string
+        options:
+          - 'specs'
+          - 'sdk'
+        default: 'sdk'
+      specs_url:
+        description: 'Run tests based on specs'
+        required: false
+        type: string
+        default: ''
+      sdk_version:
+        description: 'Run tests based on SDK'
+        required: false
+        type: string
+        default: 'LATEST'
+      sdk_namespace:
+        description: 'SDK to test'
+        required: true
+        type: string
+        options:
+          - rapid
```

Comment:
Do we need to limit the namespace here? this will limit the reusability of the action.

Reply:
While it could limit the reusability of the action, for the time being, I prefer to have a defensive approach, especially since all the work done here can change based on future circumstances.